app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -14,26 +13,14 @@ export default class AMRHeatmapVis extends React.Component {
     super(props);
 
     this.state = {
-      viewLevel: VIEW_LEVEL_GENES,
-      metric: "coverage",
+      loading: true,
```

Comment:
Don't think we still need the loading code here anymore, since this component isn't displayed until the data is ready.